src/Service/Telemetry/TelemetryEndpoint.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Service\Telemetry;
+
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class TelemetryEndpoint implements Endpoint
+{
+    private HttpClientInterface $client;
+
+    public function __construct(HttpClientInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    public function send(string $userAgent, Entry $entry): void
+    {
+        $response = $this->client->request(
+            'POST',
+            'https://telemetry.repman.io',
+            [
+                'headers' => [
+                    'Content-Type' => 'application/json',
+                    'User-Agent' => $userAgent,
```

Comment:
do we actually need this header here, all this data is included in the request anyway :thinking: